services/user.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                **kwargs) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        **kwargs
+    )
```

Comment:
Make here like in example below, because you can get two error.
1) create_user("username", "password", email=None) because email=CharField(isnull=False by default)
2) create_user("username", "password", emaill="some@gmail.com") because no exist such field 'emaill' with 'll'